redbot/core/bot.py

```diff
@@ -1305,6 +1305,25 @@ async def is_owner(self, user: Union[discord.User, discord.Member]) -> bool:
         """
         return user.id in self.owner_ids
 
+    async def get_invite_url(self) -> str:
+        """
+        Generates the invite URL for the bot.
+
+        Does not check if invites are public.
```

Comment:
It would seem there's no public API to know if the invite is public and it seems like an important part of this functionality. Could you add a method for it as well?

Reply:
Added under is_invite_url_public